x-pack/auditbeat/module/system/_meta/config.yml.tmpl

```diff
@@ -34,8 +33,3 @@
   # detect any changes.
   user.detect_password_changes: true
   {{- end }}
-  {{- if false -}}
```

Comment:
Do you want to exclude these lines from the windows reference config?

```
  # socket.state.period: 12h
  # user.state.period: 12h
```

Reply:
Agree with Andrew, for consistency with lines 15-18

Reply:
Ah yes. Pushing a commit.